src/OrchardCore.Modules/OrchardCore.AdminDashboard/Controllers/DashboardController.cs

```diff
@@ -109,9 +109,7 @@ public async Task<IActionResult> Manage()
             var dashboardCreatable = new List<SelectListItem>();
 
             var widgetContentTypes = _contentDefinitionManager.ListTypeDefinitions()
-                    .Where(t =>
-                    !string.IsNullOrEmpty(t.GetSettings<ContentTypeSettings>().Stereotype) &&
-                    t.GetSettings<ContentTypeSettings>().Stereotype.Contains("DashboardWidget"))
+                    .Where(t => t.HasStereotype() && t.GetStereotypeOrDefault().Contains("DashboardWidget"))
```

Comment:
Why you didn't use `GetStereotypeOrDefault()`

Reply:
Contains will throw exception if a null is returned.

Reply:
Anyway, I change this to use `TryGetStereotype()` instead just for simplicity.

Reply:
I mean `GetStereotype()`